test/e2e/mxGraph.model.test.ts

```diff
@@ -595,131 +601,203 @@ describe('mxGraph model', () => {
+    expectModelContainsShape('expanded_call_activity_with_loop_id', {
       kind: ShapeBpmnElementKind.CALL_ACTIVITY,
       label: 'Expanded Call Activity With Loop',
       markers: [ShapeBpmnMarkerKind.LOOP],
     });
-    expectModelContainsShape('expanded_callActivity_with_sequential_multi_instance', {
+    expectModelContainsShape('expanded_call_activity_with_sequential_multi_instance_id', {
       kind: ShapeBpmnElementKind.CALL_ACTIVITY,
       label: 'Expanded Call Activity With Sequential Multi-instance',
       markers: [ShapeBpmnMarkerKind.MULTI_INSTANCE_SEQUENTIAL],
     });
-    expectModelContainsShape('expanded_callActivity_with_parallel_multi_instance', {
+    expectModelContainsShape('expanded_call_activity_with_parallel_multi_instance_id', {
       kind: ShapeBpmnElementKind.CALL_ACTIVITY,
       label: 'Expanded Call Activity With Parallel Multi-instance',
       markers: [ShapeBpmnMarkerKind.MULTI_INSTANCE_PARALLEL],
     });
 
     // Collapsed
-    expectModelContainsShape('collapsed_callActivity_1', { kind: ShapeBpmnElementKind.CALL_ACTIVITY, label: 'Collapsed Call Activity' });
-    expectModelContainsShape('collapsed_callActivity_with_loop', {
+    expectModelContainsShape('collapsed_call_activity_id', { kind: ShapeBpmnElementKind.CALL_ACTIVITY, label: 'Collapsed Call Activity' });
+    expectModelContainsShape('collapsed_call_activity_with_loop_id', {
       kind: ShapeBpmnElementKind.CALL_ACTIVITY,
       label: 'Collapsed Call Activity With Loop',
       markers: [ShapeBpmnMarkerKind.LOOP, ShapeBpmnMarkerKind.EXPAND],
     });
-    expectModelContainsShape('collapsed_callActivity_with_sequential_multi_instance', {
+    expectModelContainsShape('collapsed_call_activity_with_sequential_multi_instance_id', {
       kind: ShapeBpmnElementKind.CALL_ACTIVITY,
       label: 'Collapsed Call Activity With Sequential Multi-instance',
       markers: [ShapeBpmnMarkerKind.MULTI_INSTANCE_SEQUENTIAL, ShapeBpmnMarkerKind.EXPAND],
     });
-    expectModelContainsShape('collapsed_callActivity_with_parallel_multi_instance', {
+    expectModelContainsShape('collapsed_call_activity_with_parallel_multi_instance_id', {
       kind: ShapeBpmnElementKind.CALL_ACTIVITY,
       label: 'Collapsed Call Activity With Parallel Multi-instance',
       markers: [ShapeBpmnMarkerKind.MULTI_INSTANCE_PARALLEL, ShapeBpmnMarkerKind.EXPAND],
     });
 
     // activity
     // Task
-    expectModelContainsShape('task_1', {
-      kind: ShapeBpmnElementKind.TASK,
-      font: {
-        isBold: false,
-        isItalic: false,
-        isStrikeThrough: false,
-        isUnderline: true,
-        name: 'Arial',
-        size: 11.0,
-      },
-      label: 'Task 1',
-    });
-    expectModelContainsShape('task_with_loop', {
+    expectModelContainsShape('task_id', { kind: ShapeBpmnElementKind.TASK, label: 'Task' });
+    expectModelContainsShape('task_with_loop_id', {
```

Comment:
:question: do you remember why you removed the expectation about font here?

Reply:
To avoid to duplicate the same expectation about font (and configuration on diagram), I just check it on **task_with_flows_id** task.